ledger/participant-integration-api/src/test/lib/scala/platform/indexer/ha/IndexerStabilityTestFixture.scala

```diff
@@ -94,6 +93,7 @@ object IndexerStabilityTestFixture {
                       }
                     )
                     .acquire()
+                  metrics = new Metrics(new MetricRegistry)
```

Comment:
Why is this change required?

Reply:
I'm actually surprised it wasn't required before actually. Using the same `MetricRegistry` instance for multiple indexers means that multiple indexers try to register a gauge under the same name (the crash/exception was triggered by InMemoryStateUpdater.scala#97). I did a quick search of what might have changed in Dropwizard Metrics between the two releases, but I would expect it to fail already actually.

Reply:
Surprised why does it even pass though. 
I've created this issue for the newly added gauges in dropwizard https://github.com/dropwizard/metrics/issues/2920, but shouldn't this fail as well during indexer restarts? We're recreating the `DbDispatcher` as part of the restart therefore it would re-create the thread pool associated with it. And the `gauge` registration was added as part of metric instrumentation for the `InstrumentedExecutorService` for the `ThreadPoolExecutor` in this case.

Reply:
This is no longer relevant since InstrumentedExecutorService does not use dropwizard anymore.